ale_linters/yaml/actionlint.vim

```diff
@@ -21,15 +21,27 @@ endfunction
 function! ale_linters#yaml#actionlint#Handle(buffer, lines) abort
     " Matches patterns line the following:
     ".github/workflows/main.yml:19:0: could not parse as YAML: yaml: line 19: mapping values are not allowed in this context [yaml-syntax]
-    let l:pattern = '\v^.*:(\d+):(\d+): (.+) \[(.+)\]$'
+    let l:pattern = '\v^.{-}:(\d+):(\d+): (.+) \[(.+)\]$'
```

Comment:
I made the regexp greedy, as it was only grabbing the last part of the error message (shellcheck), swallowing the initial and correct column/line